src/Symfony/Component/Serializer/Attribute/Ignore.php

```diff
@@ -20,7 +20,7 @@
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
 #[\Attribute(\Attribute::TARGET_METHOD | \Attribute::TARGET_PROPERTY)]
-final class Ignore
```

Comment:
Please add the `@final` annotation so that we don't support BC for child classes (as they might not work when reading attributes anyway)

Reply:
Fair enough, should be noted that many other attributes (ie. in DependencyInjection) are also not marked final. Out of scope for this PR to look into that.

Reply:
For some of them, it is intended as we explicit added support for extending them.